lib/rubocop/cop/rspec/before_after_all.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that before/after(:all) isn't being used.
+      # This leads to flaky tests when being split out.
```

Comment:
It applies in non-parallel tests too, e.g. if you forget an `after(:all)` to clean up data, then it may stick around between test runs.